optaplanner-core/src/test/java/org/optaplanner/core/impl/testdata/domain/comparable/TestdataEntityWithDifficultyComparator.java

```diff
@@ -0,0 +1,55 @@
+package org.optaplanner.core.impl.testdata.domain.comparable;
+
+import org.optaplanner.core.api.domain.entity.PlanningEntity;
+import org.optaplanner.core.api.domain.variable.PlanningVariable;
+import org.optaplanner.core.impl.domain.entity.descriptor.EntityDescriptor;
+import org.optaplanner.core.impl.domain.solution.descriptor.SolutionDescriptor;
+import org.optaplanner.core.impl.domain.variable.descriptor.GenuineVariableDescriptor;
+import org.optaplanner.core.impl.testdata.domain.TestdataObject;
+import org.optaplanner.core.impl.testdata.domain.TestdataValue;
+
+@PlanningEntity(difficultyComparatorClass = TestdataCodeComparator.class)
+public class TestdataEntityWithDifficultyComparator extends TestdataObject {
```

Comment:
This naming is confusing. It ends with "Comparator", so is it a Comparator?
No, it's a entity. See other comment for rename suggestion.